channels/us.m3u

```diff
@@ -5303,7 +5303,7 @@ https://linear-44.frequency.stream/dist/localnow/44/hls/master/playlist.m3u8
 https://linear-44.frequency.stream/dist/plex/44/hls/master/playlist.m3u8
 #EXTINF:-1 tvg-id="RevryNews.us" tvg-name="Revry News" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/AdIThwq.png" group-title="Lifestyle",Revry News
 https://revry-revrynews-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="RevryQueer247.us" tvg-name="Revry Queer 2/47" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",Revry Queer 2/47
+#EXTINF:-1 tvg-id="RevryQueer247.us" tvg-name="Revry Queer 2/47" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",Revry Queer 24/7
```

Comment:
2/47 ?

Reply:
I took it off completely, suggested by Dum4G.